app/modules/main/directives/mdtTableDirective.js

```diff
@@ -220,19 +220,33 @@
                         var body = [];
                         var customCell = [];
 
+                        // Use plain JS to append content
                         _.each(clone, function (child) {
-                            var $child = angular.element(child);
 
-                            if ($child.hasClass('theadTrRow')) {
-                                headings.push($child);
-                            } else if($child.hasClass('customCell')) {
-                                customCell.push($child);
+                            if ( child.classList !== undefined ) {
```

Comment:
the problem with `classList` is that IE 8 is not supported, and IE 11 has only partial support.
Opera Mini doesn't have any support at all. 

Can we change this to a compatible solution?

http://caniuse.com/#search=classList